<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_Title_Filter</class>
 <widget class="QWidget" name="UI_Title_Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>66</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>66</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>568</width>
    <height>66</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="titleEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::LineWrapMode::NoWrap</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Enter">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>確認</string>
   </property>
  </widget>
  <widget class="QLabel" name="textlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>例如有視窗標題為&quot;AAA BBBSSS歌名 - VLC 媒體播放器&quot;
而您想顯示&quot;BBBSSS歌名&quot;，則請只將&quot;BBBSSS歌名&quot;留下。
                                              請注意，無法顯示 &quot;AAA 歌名&quot; 或者 &quot;BBB歌名&quot;</string>
   </property>
   <property name="text">
    <string>請把要顯示的標題保留下來(滑鼠指我顯示更多資訊)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
